app/views/external_users/claims/interim_fee/_summary.html.haml

```diff
@@ -13,83 +13,79 @@
   - else
     - fee = present(claim.interim_fee)
       - unless fee.is_disbursement?
-      %table.summary
-        %caption
-          %span.table-caption.visuallyhidden
-            = t('.caption')
-        %tbody
-          %tr
-            %th{ class: 'bold' }
-              = t('shared.summary.fee_type')
-            %td
-              = fee.fee_type.description
+      %dl.govuk-summary-list
+        .govuk-summary-list__row
+          %dt.govuk-summary-list__key
+            = t('shared.summary.fee_type')
+          %dd.govuk-summary-list__value
+            = fee.fee_type.description
 
-          - unless fee.is_interim_warrant?
-            %tr
-              %th{ class: 'bold' }
-                = t('shared.summary.ppe_total')
-              %td
-                = fee.quantity
+        - unless fee.is_interim_warrant?
```

Comment:
Maybe not for this PR but there seems to be some duplication in the views that could be added to partials with params to dry things out a bit.

Reply:
I agree, there are areas for improvement, however I am currently focused on the immediate tasks.